.dependabot/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 1
+update_configs:
+  - package_manager: "java:maven"
+    directory: "/"
+    update_schedule: "weekly"
+    default_reviewers:
+      - "oleg-nenashev"
+      - "runzexia"
```

Comment:
I have never seen consent from @runzexia to be a plugin maintainer, i suggest removing him for now

Reply:
Okay!